aether-kernel/aether/kernel/api/migrations/0016_auto_20180907_1214.py

```diff
@@ -113,9 +128,4 @@ class Migration(migrations.Migration):
             model_name='mappingset',
             index=models.Index(fields=['-modified'], name='kernel_mapp_modifie_46a12a_idx'),
         ),
```

Comment:
why are you rewriting the migration file?

Reply:
Because due to the `migrate_current_mappings_to_mappingsets` method being ran. The migration order had to be resorted